model/project.go

```diff
@@ -625,15 +625,48 @@ func (p *Project) GetVariantMappings() map[string]string {
 	return mappings
 }
 
+// GetVariantsWithTask returns the names of all buildvariants that contain a certain task
```

Comment:
Tiniest nit ever: I really don't like using line breaks to indicate semantics in a comment. Go's convention is something like, "break lines if they get too long." It would be better if this were one sentence or two. I misread this as "that contains a certain task searches...."